src/api.rs

```diff
@@ -126,7 +115,7 @@ pub extern "C" fn smashline_install_acmd_script(
     priority: Priority,
     function: unsafe extern "C" fn(&mut L2CAgentBase),
 ) {
-    smashline_install_acmd_script_costume(agent, Costume { min: -1, max: -1 }, script, category, priority, function);
+    smashline_install_acmd_script_costume(agent, Costume { data: std::ptr::null(), len: 0 }, script, category, priority, function);
```

Comment:
Could this not be `Costume::from_vec([])` or something like that? I would prefer not to set a struct's internal data directly like this outside of its own internal methods.